### PR TITLE
updated guidelines to reflect boto3 preference

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```
2.3
```

##### SUMMARY

```

```

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/GUIDELINES.md [hunk 1]
```diff
@@ -4,16 +4,11 @@
 
 Since Ansible 2.0, it is a requirement that all new AWS modules are written to use boto3.
 
-Prior to 2.0, modules may of been written in boto or boto3. Modules written using boto can continue to be extended using boto.
-
-Backward compatibility of older modules must be maintained.
+Prior to 2.0, modules may of been written in boto or boto3. The effort to port all modules to boto3 has begun.
```
Comment (willthames) [inline lib/ansible/modules/cloud/amazon/GUIDELINES.md]: 'may have' - an existing error, but worth fixing.

**Comment by willthames** [comment]
shipit

**Review by dharmabumstead — APPROVED** [review]
Approve with edits.

**Review by ryansb — APPROVED** [review]

**Comment by ryansb** [comment]
Closing and reopening to trigger shippable tests (due to needs_ci label).

**Pull request merged**